homeassistant/components/backup/__init__.py

```diff
@@ -49,6 +50,7 @@
     "BackupAgent",
     "BackupAgentError",
     "BackupAgentPlatformProtocol",
+    "BackupAgentUnreachableError",
```

Comment:
this is needed to not violate the rule "Import should be using the component `rootPylint(W7424:hass-component-root-import)`"

Reply:
This is currently only used in tests. I think it's a leftover.

Probably use the listener to reload the backup agents of the platform and then don't return unavailable agents in `async_get_backup_agents`.

Reply:
we now check the `last_update_success` of the `coordinator_central` and also register the `async_notify_backup_listeners` on updates to the `coordinator_central` (_PR descrption has been updated, too_)